components/grid/selection/overview.md

```diff
@@ -86,17 +94,21 @@ See how the row selection modes work
 
 ## Notes
 
+## Selection in Template
+
+When using the Grid [Template](https://docs.telerik.com/blazor-ui/components/grid/templates) and you want to stop the Selection from being triggered you should add stopPropagation directive to the element
+
 ## Editing Modes
 
 In the [Incell EditMode]({%slug components/grid/editing/incell%}) selection can be applied only via a checkbox column. This is required due to the overlapping action that triggers selection and incell editing (clicking in the row).
 
 ## Observable Collections
 
-When binding the grid to an observable collection, changing the data source does not change the selected items automatically. 
+When binding the grid to an observable collection, changing the items in the data source will not change the selected items automatically.
 
 If you plan on changing the data source in your own code, you may want to clear or otherwise modify the `SelectedItems` collection to ensure the user will select only what they want from the new data.
 
-If you want to clear the data source, make sure to use its `.Clear()` method as this will notifiy the grid of the change. Creating an entirely new `ObservableCollection` will not.
+If you want to clear the data source, make sure to use its `.Clear()` method as this will notify the grid of the change. Creating an entirely new `ObservableCollection` will not.
```

Comment:
Check the behavior here. Does `.Clear()` on an observable collection clear the selected items? It is similar to removing a single item, so it might. In either case, note it in the bullet about observable data in the Data Changed section.

Reply:
Yes, when clearing the Data (OC) it clears the SelectedItems(IEnumerable) too.

Done